tests/performance-tests/src/tests/admin/import/import.test.ts

```diff
@@ -3,6 +3,7 @@ import { check, fail } from 'k6';
 import { Counter, Rate, Trend } from 'k6/metrics';
 import { Options } from 'k6/options';
 import exec from 'k6/execution';
+import { htmlReport } from 'https://raw.githubusercontent.com/benc-uk/k6-reporter/main/dist/bundle.js';
```

Comment:
This'll be cool - will we still get our CLI report too?

Reply:
Yep, the report via the CLI will work as normal